climada/util/lines_polys_handler.py

```diff
@@ -930,8 +930,16 @@ def _line_to_pnts(gdf_lines, res, to_meters):
     else:
         line_lengths = gdf_lines.length
 
+    res_check = len(line_lengths[line_lengths > 10*res])
+    if res_check > 0:
+        LOGGER.warning(
```

Comment:
Please fix the linter warning and make sure to add whitespaces at the end of each line string